src/modules/tests/pre-test-declarations/cat-a-mod2/pre-test-declarations.cat-a-mod2.reducer.ts

```diff
@@ -0,0 +1,45 @@
+import { PreTestDeclarations } from '@dvsa/mes-test-schema/categories/AM2';
+import * as preTestDeclarationActions from './pre-test-declarations.cat-a-mod2.actions';
+import { createFeatureSelector } from '@ngrx/store';
+
+export const initialState: PreTestDeclarations = {
+  insuranceDeclarationAccepted: false,
+  residencyDeclarationAccepted: false,
+  preTestSignature: '',
+  mod1CertificateNumber: '',
+};
+
+export function preTestDeclarationsCatAMod2Reducer(
```

Comment:
This is not part of this story. This should be implemented when CBT Number is implemented

None of the pre-test stuff should be dealt with here

Reply:
This doesn't implement any of the functionality but I needed to create the reducer for the typings. It currently won't work but is for compile type checks

Reply:
Okay I see, so in that case the actions can be reused and there is no need for the copied selector for now. Also the already existing files such as `pre-test-declaration.actions` reducer and selector should go into a folder named `common`